documentation/modules/post/windows/gather/credentials/redis_desktop_manager.md

```diff
@@ -0,0 +1,62 @@
+### Overview
+
+[RedisDesktopManager](https://github.com/uglide/RedisDesktopManager) RedisDesktopManager's credentials are saved in a JSON file in plaintext.
+
+
+### Setup Steps
+
+1. Download the latest installer of Redisdesktopmanager from https://github.com/uglide/RedisDesktopManager/releases.
+   But this needs to be subscribed before it can be downloaded. You can download the window version from another project.
```

Comment:
Instructions here aren't clear. Do you need to install both products? Is one meant to be used in place of the other due to activation requirements? If you could clarify this it would be appreciated.

Reply:
If you have subscribed to the first one, you can download the window program in the original project, but it is unnecessary to subscribe to one for testing. You can use the second one to replace it